examples/ResNet/Main.cpp

```diff
@@ -41,7 +41,7 @@ int main(int argc, const char* argv[]) {
         &resnet);
     wnn::GraphBuilder builder = wnn::CreateGraphBuilder(context);
     wnn::Operand output =
-        resnet.mLayout == "nchw" ? resnet.LoadNCHW(builder) : resnet.LoadNHWC(builder);
+        resnet.mLayout == "nchw" ? resnet.Load50V2Nchw(builder) : resnet.LoadNhwc(builder);
```

Comment:
@mingmingtasd Function `LoadNhwc`  is default for ResNet101V2NHWC model. Please pass the value of third parameter `is101V2` to function `LoadNhwc` by weights path:
```cpp
is101V2 = true; //if using '-m test-data/models/resnet/resnet101v2_nhwc/weights/'
is101V2 = false; // if using '-m test-data/models/resnet/resnet50v2_nhwc/weights/' 
```

Reply:
Yes, thanks, I understand, I just want to keep using the resnet101V2 nhwc as c++ example, only add resnet50v2 nhwc  into model tests. Because there is only "-l layout" for choosing model to use. @BruceDai

Reply:
I prefer to using resnet50v2 nhwc as the default one for model test and sample. It is better to be aligned with nchw model and it is more popular to use.

Reply:
> 

OK, I will change as your suggestions, thanks! @huningxin

Reply:
Already change to use resnet50v2 nhwc by default, PTAL again, thanks! @huningxin

Reply:
Why do we keep 101 model test since it's not used by examples?

Reply:
Please rename it to LoadNCHW that represent  reset50。